endpoint.go

```diff
@@ -638,7 +638,24 @@ func (ep *endpoint) assignAddress() error {
 	if err != nil {
 		return err
 	}
-	return ep.assignAddressVersion(6, ipam)
+
+	err = ep.assignAddressVersion(6, ipam)
+	if err != nil {
+		return err
+	}
+
+	// Verify Ipam driver returned valid IP addresses for this endpoint
+	list := []*net.IPNet{ep.Iface().Address()}
```

Comment:
Why don't you just check if `ep.Iface().Address() == nil || (len(n.ipamV6Info != 0 && ep.Iface().AddressIPv6() == nil`?